cognite/powerops/resync/models/v1/config_to_model/to_cogshop_model.py

```diff
@@ -163,9 +170,11 @@ def to_cogshop_asset_model(
                         transformations=[
                             _create_transformation(j, transformation_data)
                             for j, transformation_data in enumerate(json.loads(mapping.get("transformations", "")))
+                        ]
+                        + [
+                            _create_transformationV2(j, config.Transformation(**transformation_data))
+                            for j, transformation_data in enumerate(json.loads(mapping.get("transformations", "")))
                         ],
-                        retrieve=mapping.get("retrieve"),
-                        aggregation=mapping.get("aggregation"),
```

Comment:
Should these be removed? Shouldn't we keep everything as is for now and only add the new transformations?